app/javascript/components/login/components/idp-selection/auth-provider.js

```diff
@@ -1,19 +1,24 @@
-import { InteractionRequiredAuthError } from "msal/lib-commonjs/error/InteractionRequiredAuthError";
+import { InteractionRequiredAuthError } from "msal";
+import { CryptoUtils } from "msal/lib-es6/utils/CryptoUtils";
```

Comment:
This import is breaking the js unit tests. I ended up removing this import and combined it with the one above. Is there any reason that it should be imported the way you have it here?

Reply:
No, this is just what was auto imported by intellij. I'll revert it.